content/logs/processing.md

```diff
@@ -159,6 +159,30 @@ These settings:
 Give the following results:
 {{< img src="logs/processing/useragent_processor.png" alt="Useragent processor" responsive="true" popup="true">}}
 
+## Technical limits
+
+To make sure the Log Management solution functions in an optimal way we set the following technical limits and rules to your log events as well as to some product features. These have been designed so you never reach them. However, if this is the case please read the actions we enforce and please contact us if you have any suggestion.
```

Comment:
`so you never reach them` -> `so you may never reach them`?

Reply:
also double `please`

Reply:
Not sure about `However, if this is the case please read the actions we enforce and please contact us if you have any suggestion`, I'd just remove this line